.ci/magic-modules/generate-terraform.sh

```diff
@@ -43,7 +43,7 @@ bundle install
 
 # Build all terraform products
 if [ -n "$OVERRIDE_PROVIDER" ] && [ "$OVERRIDE_PROVIDER" != "null" ]; then
-  bundle exec compiler -a -e terraform -f "$OVERRIDE_PROVIDER" -o "${GOPATH}/src/github.com/$GITHUB_ORG/$PROVIDER_NAME/" -v "$VERSION"
```

Comment:
Does this need to be removed or can we explicitly specify GA for this provide? OiCS will only be generated @ beta when I add it to the Magician as well.

Reply:
It's a bit goofy - we'll have to do some plumbing when it comes time for that.  "$VERSION" here seems to actually just be ... the second half of the name of the short-dir, maybe?  In any event it's not the _correct_ value for the override providers.  :confused:

Reply:
sgtm 👍